ci/scripts/audit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-reporter-client
+  nancy build && mv build/dp-reporter-client $cwd/build
```

Comment:
should this nancy command be the same as the line previously referred to in [note1]
or perhaps:
```suggestion
  make audit
```
certainly the `mv` is not needed

Reply:
sorry, this comment stil stands

Reply:
Apologies last commit should have included these.  Git slip up.